src/client/chat.js

```diff
@@ -355,6 +355,38 @@ module.exports = function (client, options) {
       return
     }
 
+    if (message.startsWith('/') && mcData.supportFeature('useChatSessions')) {
+      const command = message.slice(1)
+
+      let acc = 0
```

Comment:
Since you're copying this from elsewhere, you should use a function instead of duplicating code